src/content/learn/start-a-new-react-project.md

```diff
@@ -24,7 +24,7 @@ If you want to build a new app or a new website fully with React, we recommend p
 npx create-next-app@latest
 </TerminalBlock>
 
-If you're new to Next.js, check out the [Next.js tutorial.](https://nextjs.org/learn/foundations/about-nextjs)
+If you're new to Next.js, check out the [Next.js documentation.](https://nextjs.org/docs)
```

Comment:
I think https://nextjs.org/learn is more suitable

Reply:
Hm yeah that might be a good idea actually! I was thinking initially that the first link provides more context but you're right this one was intended to be hands-on. Wanna make that change?

Reply:
https://github.com/reactjs/react.dev/pull/6468